lib/DDG/Spice/Pokemon/Data.pm

```diff
@@ -2,6 +2,7 @@ package DDG::Spice::Pokemon::Data;
 # ABSTRACT: Returns pokemon data from the pokeapi API
 
 use DDG::Spice;
+use String::Util qw(trim);
```

Comment:
@sahildua2305 please switch to `use Text::Trim;` -- we've got that one in production already 👍